### PR TITLE
Chg 230483 get route information for upload

### PR DESCRIPTION
**Pull request by yinkacocky**
Get route information for upload

### CODE REVIEW

File: EST.MIT.Web/Pages/create-bulk/CreateBulk/CreateBulk.razor [hunk 1]
```diff
@@ -0,0 +1,46 @@
+﻿@page "/create-bulk"
+
+<section class="page__header-wrapper govuk-!-margin-bottom-6">
+    <div class="govuk-width-container">
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-full">
+                <h2 class="page__heading govuk-heading-l">Upload Bulk Invoices</h2>
+                <p class="page__heading govuk-body">Use this service to create multiple invoices from a CSV file</p>
```
Comment (andygibbonsdefra) [inline EST.MIT.Web/Pages/create-bulk/CreateBulk/CreateBulk.razor:8]: Think this should say 'from an Excel file'

File: EST.MIT.Web/appsettings.json [hunk 2]
```diff
@@ -5,5 +5,10 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
-  "AllowedHosts": "*"
+  "ConnectionStrings": {
```
Comment (andygibbonsdefra) [inline EST.MIT.Web/appsettings.json:8]: These look like changes that shouldn't be checked in. Can you ask Faisal to review

File: EST.MIT.Web/appsettings.json [hunk 3]
```diff
@@ -1,9 +1,3 @@
 {
-  "Logging": {
-    "LogLevel": {
-      "Default": "Information",
-      "Microsoft.AspNetCore": "Warning"
-    }
-  },
-  "AllowedHosts": "*"
+
```
Comment (gldefra) [inline EST.MIT.Web/appsettings.json:2]: Can we please keep the keys and have empty values in the appsettings.

Eg.

`{
  "Logging": {
    "LogLevel": {
      "Default": "Information",
      "Microsoft.AspNetCore": "Warning"
    }
  },
  "ConnectionStrings": {
    "PrimaryConnection": ""
  },
  "InvoiceAPIBaseURI": "",
  "ApprovalAPIBAseURI": "",
  "ReferenceDataAPIBaseURI": ""
}
`

File: EST.MIT.Web/appsettings.json (hunk 2 above)
Comment (yinkacocky) [reply EST.MIT.Web/appsettings.json]: Resolved

**Review by gldefra — APPROVED** [review]

File: EST.MIT.Web/Pages/create-bulk/CreateBulk/CreateBulk.razor (hunk 1 above)
Comment (yinkacocky) [reply EST.MIT.Web/Pages/create-bulk/CreateBulk/CreateBulk.razor]: Resolved

File: EST.MIT.Web/appsettings.json (hunk 2 above)
Comment (andygibbonsdefra) [reply EST.MIT.Web/appsettings.json:8]: @yinkacocky , I think you need to take these settings out completely, not just clear the values assigned to them.

**Pull request merged**